Classes/Backend/Form/Element/InputText.php

```diff
@@ -48,7 +48,6 @@ public function render() {
 	public function renderHtml($name, $value, $options) {
 		$width = $options['width'];
 		// TODO: das Feld beachten!
-	 	$maxlength = $options['maxlength'];
```

Comment:
@ Better implement a TODO. This code is importend for the todo and should not be removed!

I am not a frind of automated code cleanup, a lot of necessary code can be removed.